lib/order.rb

```diff
@@ -0,0 +1,24 @@
+class Order
+  attr_reader :ordered_items, :total, :menu_dishes
+  def initialize 
+    menu = Menu.new
```

Comment:
Creating an instance of another object from inside of the initialize method like this sets up a very hard dependency between the tow objects, and makes it very difficult to properly isolate your unit tests.

You can get past this problem by making use of dependency injection:

```ruby
def initialize(menu = Menu.new)
  @menu = menu 
end
```